File: Service/OrderInvoiceService/Models/API/Products/Requests/AttributesForProductRequest.cs
// ***********************************************************************
// Assembly         : OrderInvoiceService
// Author           : Admin
// Created          : 02-22-2021
//
// Last Modified By : Admin
// Last Modified On : 02-22-2021
// ***********************************************************************
// <copyright file="AttributesForProductRequest.cs" company="OrderInvoiceService">
//     Copyright (c) . All rights reserved.
// </copyright>
// <summary></summary>
// ***********************************************************************
namespace OrderInvoiceService.Models.API.Products.Requests
{
    /// <summary>
    /// Class AttributesForProductRequest.
    /// </summary>
    public class AttributesForProductRequest
    {
        /// <summary>
        /// Gets or sets the product.
        /// </summary>
        /// <value>The product.</value>
        public ProductModel Product { get; set; }
        /// <summary>
        /// Gets or sets the attributes.
        /// </summary>
        /// <value>The attributes.</value>
        public AttributeModel[] Attributes { get; set; }
    }
}

File: DevUIArea/angular-product-setup/src/commonComponents/models/requests/products/attributesForProductRequest.ts
import { AttributeModel } from '../../products/attributeModel';
import { ProductModel } from '../../products/productModel';


export class AttributesForProductRequest {
    public product: ProductModel;
    public attributes: AttributeModel[];

    constructor() {
        this.product = new ProductModel();
        this.attributes = new Array<AttributeModel>();
    }
}